src/main/java/juja/microservices/gamification/service/KeeperService.java

```diff
@@ -0,0 +1,9 @@
+package juja.microservices.gamification.service;
```

Comment:
we didn't use interfaces for services and so on in this app, decided it isn't necessary yet because we have only one implementation. Please adhere to the policy (one style for the app).